clientd3d/music.c

```diff
@@ -145,23 +154,66 @@ DWORD OpenMidiFile(const char *lpszMIDIFileName)
       debug(("Unable to get current directory!\n"));
 
    sprintf(filename, "%s%s", current_dir, lpszMIDIFileName);
-   debug(("music filename = %s\n", filename));
-
+   debug(("music filename = %s \n", filename));
+   // Is it a background music or gameplay element music file?
+   if (deviceflag == 1)
```

Comment:
Make named constants for these 1 and 2 flags.  Also maybe rename the variable, since it's not a flag.